appstore/api/v1/views.py

```diff
@@ -539,7 +539,7 @@ def _get_product_providers(self, settings):
         Check for SSO defined in appstore settings.
         """
 
-        if settings.ALLOW_SAML_LOGIN:
+        if settings.ALLOW_SAML_LOGIN == "true":
```

Comment:
Is this case-sensitive?  Would be good if it was not.

Reply:
It isn't case sensitive.
https://github.com/helxplatform/appstore/blob/fa48c623792f188239ede176aad9655b7e20cc1a/appstore/appstore/settings/base.py#L65